fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -183,8 +183,13 @@ def fit(self, X, y, **kwargs):
     def predict(self, X):
         """Provide a prediction for the given input data.
 
-        Note that this is non-deterministic, due to the nature of the
-        exponentiated gradient algorithm.
+        Chooses the prediction label randomly based on the probabilities from `_pmf_predict`,
+        i.e., based on the aggregate of the weighted predictions of all underlying classifiers.
+        This is equivalent to randomly choosing an individual classifier based on the weights and
```

Comment:
An equation might be helpful.

Reply:
I think that's more appropriate for the user guides, don't you think?
In fact, if I had known we'd have user guides soon I probably wouldn't have written this here at all, so I consider this much more of a "let's document it and see where it lands eventually" sort of PR

Reply:
OK.